statistics/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -41,6 +43,10 @@ interface VariantManager {
             // the future if we can filter by app version
             Variant(key = "sc", weight = 0.0, features = emptyList(), filterBy = { isSerpRegionToggleCountry() }),
             Variant(key = "se", weight = 0.0, features = emptyList(), filterBy = { isSerpRegionToggleCountry() }),
+
+            // Fireproof experiment
+            Variant(key = "mi", weight = 0.0, features = emptyList(), filterBy = { isEnglishLocale() }),
```

Comment:
Remember to change the weight here to `1.0` :)

Reply:
I've updated the weight from 0.0 to 1.0 👍